beacon_node/beacon_chain/src/migrate.rs

```diff
@@ -723,9 +629,7 @@ impl<E: EthSpec, Hot: ItemStore<E>, Cold: ItemStore<E>> BackgroundMigrator<E, Ho
         let persisted_head = PersistedBeaconChain {
```

Comment:
We could remove this write, and probably delete the `PersistedBeaconChain` altogether.

Reply:
Fixed with 9ce4b33e4, but to fully remove `PersistedBeaconChain` we need to have a way to compute the genesis state root and genesis block root when booting from an existing DB. The issue is that while the state is in the DB it is keyed by root which we don't know.